definitions/divorce/json/CaseEventToFields/CaseEventToFields-alt-service-process-server-nonprod.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+  "LiveFrom": "25/08/2020",
+  "CaseTypeID": "DIVORCE",
+  "CaseEventID": "solicitorApplyForDN",
+  "CaseFieldID": "LabelContinueBecauseOfServedByProcessServer",
+  "PageFieldDisplayOrder": 1,
+  "DisplayContext": "READONLY",
+  "PageID": "dnreviewaos",
+  "PageLabel": "Review Acknowledgement of Service",
+  "PageDisplayOrder": 1,
+  "PageColumnNumber": 1,
+  "FieldShowCondition": "ServedByProcessServer=\"Yes\"",
+  "ShowSummaryChangeOption": "No"
+  },
+  {
+    "LiveFrom": "25/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorApplyForDN",
+    "CaseFieldID": "ServedByProcessServer",
+    "PageFieldDisplayOrder": 3,
+    "DisplayContext": "READONLY",
+    "PageID": "dnreviewaos",
+    "PageLabel": "Review Acknowledgement of Service",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "DNApplyForDecreeNisi=\"NeverShow\"",
```

Comment:
Why does this field need to be here?

Reply:
@jvictorcf Had same question originally, was getting errors in the build. @Dizrupto had explained its a CCD behaviour, and we need to fined this also. Do so and build went green.